### DISCUSSION TITLE
Welcome to pimd Discussions!

### DISCUSSION THREAD

**Discussion by troglobit** [troglobit/pimd#188] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eriksl** [comment]
Hi All,

I am Erik Slagter, from the Netherlands. In my daily job I am an networking administrator in a hospital, have done that for some 20 years now. Working in a hospital means being very careful and conservative with the network, so the experimenting should be done at home ;-) (and yes I do!).

At the moment my home network consists of three components: a wireless part consisting of four Aruba "Instant" access points (so one of them is a sort of controller), a firewall running on Linux on an old laptop, using core nft and a Juniper EX3300 switch that I use for routing only, no switching is taking place there. This means I need to use VRF's to keep the traffic separated from different security zones and that works quite well. The firewall is the "leakage" point between the security zones.

All wired clients have their own gw and L3 interface, all wireless clients get a unique vlan assigned that also have their L3 interface on the switch, so they are also routed on the switch. I realise this is not the "usual" situation but I don't like L2 at all.

Another hobby is programming ESP8266's (microcontrollers with wlan access) and they're, unlike the other hosts, all together in one vlan that is connected one SSID. So that's more like the common situation. It all works like a charm, but now I want to have multicast working like I had it working earlier, before I went changing everything to L3. What I exactly am trying to achieve and some context I will add in a separate thread. Pim(d) may be the solution here or it may not, I am not sure yet. All I know it doesn't work now, with or without pimd.

Thanks for your interest!
